asdf-standard

```diff
@@ -1 +1 @@
-Subproject commit f6421d381ba2b6308259b781de6434f05c879338
+Subproject commit cf9f9b8ac4d7a830f12fed5bc64f3078d5e31b9a
```

Comment:
I think Travis is failing because this points to one commit from the end of your nden/asdf-standard/add-polynomials branch, rather than 300f62, which has the fixes for the examples.